SEM/datamodels/main.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+import dlite
+
+thisdir = Path(__file__).resolve().parent
+dlite.python_storage_plugin_path.append(thisdir  / "plugins")
+
+# ThermoFisher
+fname1 = Path("example_data/(ThermoFisher) pos1_01_grid_200x.tif")
```

Comment:
For people working on Windows, this is safer:
```suggestion
fname1 = Path("example_data") / "(ThermoFisher) pos1_01_grid_200x.tif"
```

Reply:
I have verified that these give the same in Python 3.12 on Windows and Linux so I don't think this is needed when using the Path class